sw/device/tests/dif_gpio/dif_gpio_test.cc

```diff
@@ -0,0 +1,85 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "mock_mmio.h"
+
+extern "C" {
+#include "gpio_regs.h"
+#include "sw/device/lib/dif/dif_gpio.h"
+}
+
+// This is a temporary measure to work around the dependency on
+// |GPIO0_BASE_ADDR| in |GPIO_*()| family of macros in gpio_regs.h.
+// See lowrisc/opentitan#1118.
+static const uintptr_t GPIO0_BASE_ADDR = 0x0U;
```

Comment:
In C++, prefer `constexpr` over `const`; also, perfer anonymous namespaces over `static`.

Reply:
Absolutely! I copied this over from `dif_gpio.c` and didn't take a second look since I expect this part to be removed soon. I will minimize to only define one offset.